typescript/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Why is this a new file? Your `GoodMorningAli/porting-typescript...` library should already have a `package.json`; I created it on Friday

Reply:
This is the kurtosis-testsuite-starter-pack repo, you set everything up inside kurtosis-testsuite-api-lib on Friday if I'm not mistaken